src/pages/VerifySettingsPage/VerifySettingsPage.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useCallback } from "react";
+import { useDeskproAppEvents, useDeskproAppClient } from "@deskpro/app-sdk";
+import { getAccounts } from "../../api/api";
+// import { getError } from "../../utils";
+import { VerifySettings } from "../../components/VerifySettings";
+import type { FC } from "react";
+import type { Maybe, Settings } from "../../types";
+import type { IAccount } from "../../api/types";
+
+const getError = (err: Maybe<Error>): string => {
+  const defaultError = "Failed to connect to Wrike, settings seem to be invalid";
+  let error;
+
+  if (!err) {
+    error = defaultError;
+  } else {
+    try {
+
+      const parsedError = JSON.parse(err.message);
+      const parsedMessage = JSON.parse(parsedError.message);
+      error = parsedMessage.errorDescription || defaultError;
+    } catch (e) {
+      error = defaultError;
+    }
+  }
+
+  return error;
+};
+
+const VerifySettingsPage: FC = () => {
+  const { client } = useDeskproAppClient();
+  const [accounts, setAccounts] = useState<Maybe<Array<IAccount["name"]>>>(null);
+  const [settings, setSettings] = useState<Settings>({});
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<Maybe<string>>(null);
```

Comment:
Avoid `Maybe` here as it's always a `string` or` null` so use `string|null` instead to be more accurate.

Reply:
Because `Maybe` it’s about the possible presence of an error.

Reply:
Maybe we have an error here